src/components/Table/GridTable.tsx

```diff
@@ -944,40 +888,56 @@ function GridRow<R extends Kinded, S>(props: GridRowProps<R, S>): ReactElement {
+    (style.nestedCards?.firstLastColumnWidth ?? 0) - openCardWidth - (leafCardStyles?.bColor ? 1 : 0);
 
   const rowStyleCellCss = maybeApplyFunction(row, rowStyle?.cellCss);
   const rowCss = {
-    // For virtual tables use `display: flex` to keep all cells on the same row, but use `flexNone` to ensure the cells stay their defined widths
-    ...(as === "table" ? {} : Css.df.addIn("&>*", Css.flexNone.$).$),
+    // For virtual tables use `display: flex` to keep all cells on the same row. For each cell in the row use `flexNone` to ensure they stay their defined widths
+    ...(as === "table" ? {} : Css.relative.df.fg1.fs1.addIn("&>*", Css.flexNone.$).$),
     ...((rowStyle?.rowLink || rowStyle?.onClick) &&
       style.rowHoverColor && {
         // Even though backgroundColor is set on the cellCss (due to display: content), the hover target is the row.
         "&:hover > *": Css.cursorPointer.bgColor(maybeDarken(rowStyleCellCss?.backgroundColor, style.rowHoverColor)).$,
       }),
     ...maybeApplyFunction(row, rowStyle?.rowCss),
+    // Maybe add the sticky header styles
     ...(isHeader && stickyHeader ? Css.sticky.top(stickyOffset).z1.$ : undefined),
+    // Card styles apply a calculated padding to ensure the content lines up properly across all columns
+    ...(currentCardPaddingWidth ? Css.pxPx(currentCardPaddingWidth).$ : undefined),
+    // Leaf cards define their own borders + padding
+    ...(leafCardStyles
+      ? // We can have versions of the same card as a leaf and not as a leaf.
+        // When it is not a leaf then it has chrome rows that create the top and bottom "padding" based on border-radius size. (brPx = "chrome" row height)
+        // When it is a leaf, then we need to apply the brPx to the row to ensure consistent spacing between leaf & non-leaf renders
+        // Additionally, if the leaf card has a border, then subtract the 1px border width from the padding to keep consistent with the "chrome" row
+        Css.pyPx(leafCardStyles.brPx - (leafCardStyles.bColor ? 1 : 0))
+          .borderRadius(`${leafCardStyles.brPx}px`)
+          .bgColor(leafCardStyles.bgColor)
+          .if(!!leafCardStyles.bColor)
+          .bc(leafCardStyles.bColor).ba.$
+      : undefined),
   };
 
-  const Row = as === "table" ? "tr" : "div";
-
-  const openCardStyles =
-    typeof openCards === "string"
-      ? openCards
-          .split(":")
-          .map((openCardKind) => style.nestedCards!.kinds[openCardKind])
-          .filter((style) => style)
-      : undefined;
-  const currentCard = openCardStyles && openCardStyles[openCardStyles.length - 1];
   let currentColspan = 1;
-  const maybeStickyHeaderStyles = isHeader && stickyHeader ? Css.sticky.top(stickyOffset).z1.$ : undefined;
 
-  return (
-    <Row css={rowCss} {...others}>
-      {openCardStyles &&
-        maybeAddCardPadding(openCardStyles, "first", {
-          ...maybeStickyHeaderStyles,
-          ...(columnSizes ? Css.w(columnSizes[0]).$ : {}),
-        })}
```

Comment:
Huh, that's neat that this can go away, and `maybeWrapCard` does it instead. Ah, I guess the css grid restriction going away means we don't need the cells to be exactly at the 2nd level of the DOM anymore. Nice!

...can `mayeAddCardPadding` being deleted?

Reply:
Yea, I meant to delete `maybeAddCardPadding`. Thanks for the reminder.